internal/cfgmgr/cfgmgr.go

```diff
@@ -0,0 +1,149 @@
+package cfgmgr
```

Comment:
Right now I expect this package to only be used by the device-util command. I can see why this package name would be confusing, though I would prefer not to move it to a single place in the event that in the future someone needs to expand the functions exported or include it in other places. 

I'm fine with renaming the package. Would a name like `cfgmgrdll` be okay?

Reply:
I prefer we name the package reflecting the functionality exposed (which appears to be Windows device ID querying), rather than the implementation details (that it happens to use cfgmgr32.dll). For instance, if we wanted to add additional functions to support some other device ID querying scenarios, and they happened to involve functions in another DLL, I think they should go in this package as well, but that creates confusion if the package is named for cfgmgr32.dll.

Reply:
Makes sense. My concern is that if we name this something like `windevice` and later we want to export more functions from cfgmgr into a separate package (ie functions that may not have anything to do with device ID querying), then we may be forced to export constants/functions needed across both and it's unclear where they would belong.

Reply:
From the look of the next commit, it appears you're going to be adding a bunch of Windows API bindings throughout this PR. In that case, it might be easiest to do the following:

- Make a new package in `internal/winapi` that contains the low-level syscall bindings and definitions.
- Make a new package for each piece of useful functionality that calls into the low-level bindings (e.g. `internal/windevice`).

Reply:
And maybe someday the things in `internal/winapi` could be moved into `golang.org/x/sys/windows`.

In general, though, it is fine to put syscall bindings in a specific package and keep them unexported, unless we are pretty sure we will need to call them from another package. It's also not hard to just move them out when we need them.

Reply:
> * Make a new package in `internal/winapi` that contains the low-level syscall bindings and definitions.
> * Make a new package for each piece of useful functionality that calls into the low-level bindings (e.g. `internal/windevice`).

I like this idea, may refactor the job object stuff to follow suit.

Reply:
Updated! I liked this layout much better as well.